src/apis/ValidateAADToken.ts

```diff
@@ -33,7 +33,8 @@ export class ValidateAADToken {
         let msaOpenIdMetadata = new OpenIdMetadata("https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration");
         return (req: express.Request, res: express.Response) => {
             // Get bearer token
-            let authHeaderMatch = /^Bearer (.*)/i.exec(req.headers["authorization"]);
```

Comment:
Change to req.header("authorization"), which will be of type string
(This is so we're consistent w/ the auth sample)

Reply:
Done.